source/_integrations/homematic.markdown

```diff
@@ -469,8 +469,10 @@ lock:
 
 {% endraw %}
 
-To get the current value of the current lock status, you have to create a system variable (in the example above it is `base_lock_status`) and create a program on CCU, which updates the variable with every change of the Lock level to `true` for locked and `false` for unlocked.
+Since 2021.12.0 a sensor for the lock status is available. It can be be used as seen in the value template above.
 
+Before 2021.12.0 the same could be achieved with a system variable on the CCU. Use `value_template: "{{ state_attr('homematic.ccu2', 'base_lock_status') }}"` and
+create a program on CCU, which updates the variable with every change of the lock level to `true` for locked and `false` for unlocked.
```

Comment:
We don't keep a changelog in our documentation. So, I guess this whole section can now be removed?

```suggestion
```